### PR TITLE
Update bootstrap-tourist.js

### PR DESCRIPTION
**Pull request by pau1phi11ips**
Changed default Bootstrap framework to v4

Added Tour.init() that just console.log's a notice that you shouldn't be using .init() anymore. It can be dropped in as a direct replacement for Bootstrap Tour without breaking anything that way.

### CODE REVIEW

**Comment by IGreatlyDislikeJavascript** [comment]
Thanks for the pull request, much appreciated.

I agree with the console log in init(), anything that supports drop in replacement makes perfect sense.

I'm not sure that the default framework should be changed to BS4 though, for the same reason! Original Tour was BS3 compatible only, so anyone using Tour with BS3 would expect Tourist to work with BS3 by default.

Anyone switching to Tourist for BS4 compatibility will reasonably expect to have to do "something". So it feels like BS3 should be the default.

In other words - everyone using BS3 will already have a working tour, so switching should be a drop in replacement. Anyone using BS4 can't have a working tour using original Bootstrap Tour, so I think it's reasonable that they should need to use the framework option as an additional "implementation" step.

Does that make sense?

**Comment by pau1phi11ips** [comment]
I see your point and I guess there's still more Bootstrap 3 users vs Bootstrap 4.

Original Tour was compatible with Bootstrap 4 up to version 4.1 though. Could maybe be add a build step to output bootstrap3-tourist.min.js and bootstrap4-tourist.min.js with the relevant Defaults set?

**Comment by IGreatlyDislikeJavascript** [comment]
Yes, the build step makes sense. However for now I'm only providing the plain js in this repository (not even a minified certain etc). The full reasons are explained in the read me!

So I'll add your init() pull but not include the framework change. Not sure if I can accept only part of a pull on github so if not, I'll close and manually.

Thank you for your contribution

**Comment by IGreatlyDislikeJavascript** [comment]
Manually incorporated the .init() change into Tourist v0.11. Thanks for your contribution!
